R/S3_predict.R

```diff
@@ -125,8 +125,11 @@ predict.lm_robust <- function(object,
 
     ret <- list()
 
-    var_fit <-
-      apply(X[, !beta_na, drop = FALSE], 1, function(x) tcrossprod(crossprod(x, object$vcov), x))
+    var_fit <- apply(
+      X[, !beta_na, drop = FALSE],
+      1,
+      function(x) tcrossprod(crossprod(x, object$vcov), x)
```

Comment:
double checking ,do we really want `(t(x) V) t(x)` or are you relying on implicit vector-matrix conversions.

`outer(x,x) * object$vcov` is a little more obvious if that's what you really want

Reply:
I actually want x'Vx where x is a column vector, so I'm relying on the implicit conversion.